tests/ashley/machina_extensions/forum/test_forum_view.py

```diff
@@ -364,3 +369,177 @@ def test_testing_query_param(self):
             '<a href="?o=3&whatever=2" class="toggle descending" title="Toggle sorting"></a>',
             html=True,
         )
+
+    def test_testing_topic_is_sticky_stay_sticky_default_order(self):
+        """Request page with a sticky topic and check that the sticky topic is always the first
+        result on default order"""
+        forum, url_list_topic = self._get_url_list_topic_with_three_topics()
+        # Creates a post for sticky topic
+        PostFactory(
+            topic=TopicFactory(forum=forum, type=Topic.TOPIC_STICKY),
+            subject="TOPIC STICKY ONE",
+        )
+
+        # Calls page with default order
+        url = f"{url_list_topic}"
+        response = self.client.get(url)
+
+        topicC = forum.topics.get(subject__startswith="TOPIC C the newest")
+        topicB = forum.topics.get(subject__startswith="TOPIC B the eldest")
+        topicA = forum.topics.get(subject__startswith="TOPIC A created second")
+        topicSticky = forum.topics.get(subject__startswith="TOPIC STICKY ONE")
+
+        # Controls topics have been created in the order assumed in their titles
+        self.assertTrue(
```

Comment:
If this assertion fails, it is hard to debug: pytest will just say something like "assertion is not true" and you will not know which value is incorrect.

I suggest to use `assertGreater` or `assertLess` instead to ease debug.

Reply:
I see what you are saying but that would force me to compare two by two and give more importance than needed for this test.
This test is only a casual test to give context to the reader, we don't have doubt about it, it's only for understanding as we create topics from a sub function. I get the idea of debugging but in this current context, the assertTrue seems more appropriate. WDYT ?

Reply:
I understand that the code will be less pleasant to read, but IMO this is more important to get quickly to the bug if a test is failing.

If a test doesn't bring any additional value and it is proven that it cannot fail, perhaps it has no reason to exist?

Generally, I would avoid using `assertTrue/assertFalse` except if i really want to check a variable containing a boolean value. All tests can be written with these 2 assertions, but it's really painful to debug.

Reply:
It gives understanding as explained above, I wouldn't have done this test if I wasn't using a function to create my post and topics, it's for the user that read not to have to worry and give the context directly in one look.
If i break it in 5 lines, imo it will be less obvious that we are only giving context, we are testing below one by one and that's what really matters in this test.
So to me it's not a matter that the code would be less pleasant, it's a matter of relevance, giving 5 lines on that makes it important where it's not, it's just to inform, ok they have been created away in another function but look, they are ordered  the same way they are call and one line for reassurance is good.
That's my last shot, after I'll change it :-)

Reply:
Anyway it's not a big deal lol we are only talking about this change

`# Controls topics have been created in the order assumed in their titles
        self.assertGreater(topicSticky.last_post_on,topicC.last_post_on)
        self.assertGreater(topicC.last_post_on,topicA.last_post_on)
        self.assertGreater(topicA.last_post_on,topicB.last_post_on)`

self.assertTrue( topicSticky.last_post_on  > topicC.last_post_on > topicA.last_post_on > topicB.last_post_on )